### ISSUE TITLE
MagiskHide invalid for the package "com.hktpayment.tapngo"

### ISSUE THREAD

**Issue by bychlz39104** [topjohnwu/Magisk#903]
Magisk: 18.1-b245931c(18002)
MagiskManager: uninstalled
app: https://play.google.com/store/apps/details?id=com.hktpayment.tapngo
Pixel2 XL PQ1A.181205.002, Dec 2018

fastboot boot boot.img
am force-stop com.hktpayment.tapngo
pm clear com.hktpayment.tapngo
pm grant com.hktpayment.tapngo android.permission.READ_PHONE_STATE
appops set com.hktpayment.tapngo READ_EXTERNAL_STORAGE ignore
launch app ✓no warning
reboot bootloader
fastboot boot patched_boot.img
pm uninstall com.topjohnwu.magisk
magiskhide --add com.hktpayment.tapngo
am force-stop com.hktpayment.tapngo
pm clear com.hktpayment.tapngo
pm grant com.hktpayment.tapngo android.permission.READ_PHONE_STATE
appops set com.hktpayment.tapngo READ_EXTERNAL_STORAGE ignore
launch app ✗warning for root
![screenshot_20181224012041](https://user-images.githubusercontent.com/46088801/50386097-7cc04e80-071b-11e9-9bd8-3b77b02f6f28.jpg)

**Comment by Didgeridoohan** [comment]
I did a quick test and the app can launch just fine on my Oneplus 3T, with Magisk 18002 installed and with the Manager repackaged (with the app on the Hide list, of course). Points to it being an issue with your device and/or setup or similar.

Is MagiskHide otherwise working as it should on your device?

Useful links:
https://didgeridoohan.com/magisk/MagiskHide#hn_Hiding_root_from_apps
https://didgeridoohan.com/magisk/MagiskHideHelp

**Comment by bychlz39104** [comment]
Of course can launch, and at the first time the app working fine. When you launch it a second time, a warning will appear(maybe need login?).
![screenshot_20181224012041](https://user-images.githubusercontent.com/46088801/50386097-7cc04e80-071b-11e9-9bd8-3b77b02f6f28.jpg)

**Comment by Didgeridoohan** [comment]
That's new info...

You only wrote "launch app x", meaning the app won't launch. That's what I tested, and without the app on the Hide list it simply refuses to open. It's good being detailed and precise when reporting about possible issues...

Try _everything_ from the first link I posted and if you still can't get things working, follow the second link and take a look at how to provide proper logs. Without logs showing that there's actually anything going on you'll very likely just see your issue being closed without further comment.
